### PR TITLE
[Doc Improvement] [Replace hardcoded hostRedirectURL snippet][3706097]

### PR DESCRIPTION
**Pull request by mobajemu-msft**
* Replace hardcoded hostRedirectURL snippet with tokenized form in Teams tabs OAuth article

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1e3cdb1d0a56f560c1f08aa58b87049c77b46f94/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Fauth-oauth-provider.md) |   |   | [link]( ) | [:warning:](# "Could not check with Acrolinx&#10;Please close and reopen pull request") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1e3cdb1d0a56f560c1f08aa58b87049c77b46f94/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Fauth-oauth-provider.md) | 95 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7bfd3639-fdcc-4746-9b20-3e0539a9dde9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by vikramtha — APPROVED** [review]
Looks good to me

**Review by erikadoyle — APPROVED** [review]

File: msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ The following image provides the flow to add authentication to external browsers
 
     ```JavaScript
        authentication.authenticate({
-          url: `${window.location.origin}/auth-start?oauthRedirectMethod={oauthRedirectMethod}&authId={authId}&hostRedirectUrl=${url}&googleId=${googleId}`,
+          url: `${window.location.origin}/auth-start?oauthRedirectMethod={oauthRedirectMethod}&authId={authId}&hostRedirectUrl={hostRedirectUrl}&googleId=${googleId}`,
```
Comment (ChetanSharma-msft) [inline msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md:52]: Is hostRedirectUrl is a place holder like authId or do we need to replace it with actual value?

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comment.

File: msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md (hunk 1 above)
Comment (mobajemu-msft) [reply msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md:52]: Yes it is a placeholder

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by ChetanSharma-msft — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**